poet/poet.py

```diff
@@ -134,10 +147,10 @@ def get_checksum_from_pip_source_file(pip_source_file: Path) -> str:
     if not pip_source_file.exists():
         raise PipSourceMetadataException("File does not exist: %s" % pip_source_file)
     
-    return sha256(pip_source_file.read_bytes()).hexdigest()
+    return sha256(pip_source_file.read_bytes(), init=False).hexdigest()
```

Comment:
i think this was meant to be on line 97?
```suggestion
    return sha256(pip_source_file.read_bytes()).hexdigest()
```